<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackagingSettingsPage</class>
 <widget class="QWidget" name="PackagingSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="indentation">
        <number>0</number>
       </property>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Stock</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="addButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/icons/add.png</normaloff>:/res/icons/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../res.qrc">
           <normaloff>:/res/icons/bin.png</normaloff>:/res/icons/bin.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="detailsWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="1">
       <widget class="QSpinBox" name="stockSpinBox">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="stockLabel">
        <property name="text">
         <string>Stock:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="restockUrlLabel">
        <property name="text">
         <string>Restock URL:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ClickyLineEdit" name="restockUrlEdit">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="placeholderText" stdset="0">
         <string/>
        </property>
        <property name="type" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickyLineEdit</class>
   <extends>QFrame</extends>
   <header>widgets/clickylineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>stockSpinBox</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
